### ISSUE TITLE
Packer Crash with vmware-iso on ESX

### ISSUE THREAD

**Issue by claudekenni** [hashicorp/packer#5933]
Packer is crashing when using ESX vmware-iso
Version: 1.2.0

Gist with Log
https://gist.github.com/claudekenni/4bf75f32c3e7fe6aa177f62bff115abd

```
○ → packer build sles12.json
vmware-iso output will be in this color.

Build 'vmware-iso' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> vmware-iso: unexpected EOF

==> Builds finished but no artifacts were created.
panic: non-positive interval for NewTicker
2018/02/22 10:16:01 packer:
2018/02/22 10:16:01 packer: goroutine 93 [running]:
2018/02/22 10:16:01 packer: time.NewTicker(0x0, 0x0)
2018/02/22 10:16:01 packer:     /usr/local/go/src/time/tick.go:23 +0x1d0
2018/02/22 10:16:01 packer: github.com/hashicorp/packer/communicator/ssh.(*comm).Start.func1(0xc42000f2f0, 0xc420114090)
2018/02/22 10:16:01 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/communicator/ssh/communicator.go:118 +0x46
2018/02/22 10:16:01 packer: created by github.com/hashicorp/packer/communicator/ssh.(*comm).Start
2018/02/22 10:16:01 packer:     /Users/mwhooker/go/src/github.com/hashicorp/packer/communicator/ssh/communicator.go:114 +0x209
2018/02/22 10:16:01 [INFO] (telemetry) ending vmware-iso
2018/02/22 10:16:01 [INFO] (telemetry) found error: unexpected EOF
2018/02/22 10:16:01 ui error: Build 'vmware-iso' errored: unexpected EOF
2018/02/22 10:16:01 Builds completed. Waiting on interrupt barrier...
2018/02/22 10:16:01 machine readable: error-count []string{"1"}
2018/02/22 10:16:01 ui error:
==> Some builds didn't complete successfully and had errors:
2018/02/22 10:16:01 machine readable: vmware-iso,error []string{"unexpected EOF"}
2018/02/22 10:16:01 ui error: --> vmware-iso: unexpected EOF
2018/02/22 10:16:01 ui:
==> Builds finished but no artifacts were created.
2018/02/22 10:16:01 [INFO] (telemetry) Finalizing.
2018/02/22 10:16:01 /usr/local/bin/packer: plugin process exited
2018/02/22 10:16:01 waiting for all plugin processes to complete...



!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.

[1]: https://github.com/hashicorp/packer/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by SwampDragons** [comment]
Thanks for reporting -- I think this is a duplicate of #5926. The good news is, we've already investigated and fixed it.  It was caused by #5830 and should be fixed by a4123ea. The bad news is, you have to wait for the 1.2.1 release, but it should be very soon. Sorry for the regression; for now, you can either revert to 1.1.3 for now or build 1.2.1-dev from source.
